Automattic/wp-calypso
# Add built by wp marketing card

#### Proposed Changes

* This PR adds a new "Built by WP" card to `/marketing/tools`

![new-built-by-wp-card](https://user-images.githubusercontent.com/140841/215909149-864084c7-6b6f-429f-8aec-2436ae14a8cf.png)


#### Testing Instructions



* Load this PR and go to `/marketing/tools`
* Ensure this card loads (see picture above)
* Ensure the card matches the Figma pebzTe-Fg-p2#comment-1122
* Ensure that clicking the button sends you to `https://builtbywp.com/?utm_campaign=theme-all&utm_medium=automattic_referred&utm_source=WordPresscom`
* Ensure the clicks are Tracked. PCYsg-cae-p2

#### Pre-merge Checklist





Related to #
